cfme/tests/services/test_service_catalogs_bundles.py

```diff
@@ -0,0 +1,144 @@
+import fauxfactory
+import pytest
+from widgetastic.utils import partial_match
+
+from cfme import test_requirements
+from cfme.infrastructure.provider.virtualcenter import VMwareProvider
+from cfme.markers.env_markers.provider import ONE_PER_TYPE
+from cfme.services.myservice import MyService
+from cfme.services.service_catalogs import ServiceCatalogs
+from cfme.utils.generators import random_vm_name
+from cfme.utils.wait import wait_for
+
+
+pytestmark = [
+    pytest.mark.meta(server_roles="+automate"),
+    pytest.mark.long_running,
+    test_requirements.service,
+    pytest.mark.tier(2),
+    pytest.mark.provider([VMwareProvider], selector=ONE_PER_TYPE, scope="module"),
+]
+
+
+@pytest.fixture(scope="function")
+def catalog_item_setups(request, appliance, provider, setup_provider_modscope, provisioning,
+                        catalog, dialog):
+    """
+    This fixture is used to create two catalog items instance pointing to method
+    """
+    template, host, datastore, vlan = list(map(
+        provisioning.get, ('template', 'host', 'datastore', 'vlan'))
+    )
+    provisioning_data = {
+        'catalog': {'catalog_name': {'name': template, 'provider': provider.name},
+                    'vm_name': random_vm_name('service')},
+        'environment': {'host_name': {'name': host},
+                        'datastore_name': {'name': datastore}},
+        'network': {'vlan': partial_match(vlan)},
+    }
+    cat_list = []
+    for _ in range(2):
+
+        catalog_item = appliance.collections.catalog_items.create(
+            provider.catalog_item_type,
+            name=fauxfactory.gen_alphanumeric(15, start="cat_item_"),
+            description="my catalog",
+            display_in=True,
+            catalog=catalog,
+            dialog=dialog,
+            prov_data=provisioning_data
+        )
+
+        cat_list.append(catalog_item)
+        request.addfinalizer(catalog_item.delete_if_exists)
```

Comment:
I would rather seen this right after the `catalog_item` is being assigned. The append on #52 probably won't fail, but I think it is good habit to have and it may mitigate chance of inserting some code between the object creation and deletion scheduling statement.